src/server_manager/install_scripts/install_server.sh

```diff
@@ -81,16 +94,92 @@ function log_for_sentry() {
 # Check to see if docker is installed.
 function verify_docker_installed() {
   if ! command_exists docker; then
-    log_error "Docker CE must be installed, please run \"curl -sS https://get.docker.com/ | sh\" or visit https://docs.docker.com/install/"
-    exit 1
+    log_error "FAILED"
+    echo -n "> Would you like to install Docker? This will run 'curl -sS https://get.docker.com/ | sh'. [Y/n] "
+    if ! run_step_with_user_confirmation "Installing Docker" install_docker; then
```

Comment:
This command is confusing to me. When it fails, was it the confirmation, or the step itself?

I think the suggestion I gave for separating the prompt makes it clearer

Reply:
Done in 2abeb5d.